lib/jekyll-titles-from-headings/generator.rb

```diff
@@ -2,9 +2,13 @@ module JekyllTitlesFromHeadings
   class Generator < Jekyll::Generator
     attr_accessor :site
 
-    TITLE_REGEX = %r!\A\s*\#{1,3} (.*)\n$!
+    TITLE_REGEX = %r!\A\s*\#{1,3}\s+(.*)\n$!
```

Comment:
The code I’m changing already required _one_ space. This just allows for more than one to be used.

(I just saw this in passing, could drop it from the PR if you prefer.)